src/syft/grid/duet/__init__.py

```diff
@@ -135,6 +147,8 @@ def launch_duet(
     print("♫♫♫ >")
```

Comment:
can we move the print statement for the network_url to be after it finds it?

Reply:
basically i want both clients to be able to know what network url they're using but this prints it first.

Reply:
Looks good